### PR TITLE
Enhances People List styling and fixes bug

### PR DESCRIPTION
**Pull request by mk4111**
See #666 
See #681 
See #687 
See #697 

- Unifies the Explore (Orgs, Chals, Insights), People List and Resources List styling (see more in #681 )
- The styling in #697 applied to #666 and these have thus been grouped together.
- The issue in #687 about overflowing text in the email field of People List has also been rectified.

# Merge after #685

### CODE REVIEW

File: templates/partials/user.html [hunk 1]
```diff
@@ -33,14 +34,14 @@
             id="{{id}}">
 
     <div class="list__hidden-details">
-      <a class="list__data" href="mailto:{{email}}">
-        <span>{{email}}</span>
+      <a class="list__data u-teal" href="mailto:{{email}}">
+        <div class="list_-data u-teal u-text-wrap">{{email}}</div>
```
Comment (harrygfox) [inline templates/partials/user.html]: should be list__data I guess :)

**Comment by harrygfox** [comment]
@mk4111 back to you to fix conflicts and little one in user.html

File: templates/partials/user.html (hunk 1 above)
Comment (mk4111) [inline templates/partials/user.html]: Oooopth! Thanks for catching that!!!

**Comment by harrygfox** [comment]
Ready to be merged once conflicts are resolved @mk4111

**Comment by harrygfox** [comment]
yeah same again @mk4111 conflictatores

**Pull request merged**